### PR TITLE
Add some more questions/sections

### PR DESCRIPTION
**Pull request by jotaen**
Some ideas for further questions that came to my mind. Feel free to take them over (or disregard) as you like 😄

### CODE REVIEW

File: questionnaires/webops.md [hunk 1]
```diff
@@ -20,11 +21,23 @@
 - What is ports and what are they used for
 - How many ports
 - What is a TCP router?
+- How is TCP different from UDP?
+- What is congestion control?
```
Comment (bsingr) [inline questionnaires/webops.md:25]: @jotaen I'd see congestion control as advanced topic, agree?

 i need this material / questions to be effective and practical for teaching beginners in web ops or intermediate web devs. i think i will create a separate question sets for beginners / advanced.

File: questionnaires/webops.md [hunk 2]
```diff
@@ -59,6 +72,10 @@
 1. Can you sketch a raw http request and response?
 2. What is HTTPS, SSL and TLS?
 3. How sketch the steps how do DNS, TLS and HTTP work together when a user browses https://example.com?
+4. Can you explain what the resource model is in REST?
+5. What is QUIC?
+6. What fragments/parts does the following URL consist of? https://www.example.org:443/search?query=Foo#edit
+7. If you intercept an HTTPS request on the network, which information can you derive?
```
Comment (bsingr) [inline questionnaires/webops.md:78]: @jotaen just love this question, highly compressed knowledge :)

File: questionnaires/webops.md (hunk 1 above)
Comment (jotaen) [reply questionnaires/webops.md:25]: To be fair, I happened to see that you created this repo last week, and I was just in the mood to dump some questions that came to my mind spontaneously. So feel free to take or leave what you like 😄  (Updated PR description)

But regarding your question, I’d agree that congestion control is more advanced.

**Comment by jotaen** [comment]
Howdy! I’m closing this as I’m doing some spring-cleaning in my Github account. But feel free obviously to re-open or do whatever you want with this branch 😸

**Pull request closed without merging**